packages/MarkdownEditor-Tests.package/MarkdownDelimiterEmphasisTest.class/instance/testMatchesOpener.st

```diff
@@ -0,0 +1,14 @@
+tests - matching
+testMatchesOpener
```

Comment:
renaming the test cases to their assert-statements would help realising what you're testing, so here that would be 'testOpenerAndCloserMatchesOpener'